### ISSUE TITLE
Exported files have the `.pem` extension but the they are in `der` format

### ISSUE THREAD

**Issue by jimmykarily** [Itxaka/keydump#2]
With `DB-Microsoft_Corporation_UEFI_CA_2011-5.pem` being one of the exported files, this command fails:

```
openssl verify DB-Microsoft_Corporation_UEFI_CA_2011-5.pem
```

while this works:

```
openssl x509 -inform der -in DB-Microsoft_Corporation_UEFI_CA_2011-5.pem -text -noout
```

I guess the exported format is the `der` one, not `pem` (which would be a plain text if I'm not mistaken).

**Comment by Itxaka** [comment]
good catch. Noticed that this repo is basically an example on how to use the new sdk functions to extract the keys, not mean to be consumed other than for testing before integrating this into whatever place we want it.

I'll fix it meanwhile so the example work as expected :D
